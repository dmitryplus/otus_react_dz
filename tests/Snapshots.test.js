import React from 'react';

describe('Snapshot renders', () => {
    // test('Calculator', () => {
    //     const component = renderer.create(<Calculator result={'empty'} />);
    //     let tree = component.toJSON();
    //     expect(tree).toMatchSnapshot();
    // });
    //
    // test('Button', () => {
    //     const component = renderer.create(<ScaleBlock label={'0'} />);
    //     let tree = component.toJSON();
    //     expect(tree).toMatchSnapshot();
    // });
});
