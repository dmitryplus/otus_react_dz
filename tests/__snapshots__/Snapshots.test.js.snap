// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot renders Button 1`] = `
<button
  onClick={[Function]}
  type="button"
>
  0
</button>
`;

exports[`Snapshot renders Calculator 1`] = `
<div>
  <div
    className="buttons-row"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      9
    </button>
  </div>
  <div
    className="buttons-row"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      6
    </button>
  </div>
  <div
    className="buttons-row"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      3
    </button>
  </div>
  <p
    data-testid="calculator-result"
  >
    empty
  </p>
</div>
`;
